client/lib/browser-detection/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { isUndefined } from 'lodash';
+
+export const isFirefox = ! isUndefined( window.InstallTrigger );
+
+export const isIE11 = (
+	window &&
+	window.MSInputMethodContext &&
+	document &&
+	document.documentMode
+);
```

Comment:
also @mtias should be aware of this. others could easily take advantage of it and we may have different ways of accomplishing the same goals